parts/k8s/kubernetesmasterresources.t

```diff
@@ -236,9 +231,8 @@
         "name": "nicLoopNode"
       },
       "dependsOn": [
-{{if .MasterProfile.IsCustomVNET}}
```

Comment:
Before: if custom VNET depend on nsgID in master NIC / else depend on vnetID in master NIC

After: if custom VNET depend on nsgID in master NIC / else depend on vnetID + nsgID in master NIC